src/libraries/System.Security.Cryptography.Primitives/tests/HmacAlgorithmTest.cs

```diff
@@ -40,7 +40,7 @@ public void ResetAlgorithmName()
                 hmac.HashName = "SHA1";
 
                 // On desktop builds this next line will succeed (modulo FIPS prohibitions on MD5).
```

Comment:
This whole comment can be deleted. (Or changed to say "On .NET Framework..." and "On .NET Core"